packages/xrpl/src/utils/getNFTokenID.ts

```diff
@@ -15,19 +16,36 @@ interface NFToken {
   }
 }
 
+/**
+ * Ensures that the metadata is in a deserialized format to parse.
+ *
+ * @param meta - the metadata from a `tx` method call. Can be in json format or binary format.
+ * @returns the metadata in a deserialized format.
+ */
+export function ensureDecodedMeta(
```

Comment:
Why is this exported? It should just be a helper function, right?

Reply:
I needed it in my test and thought others might as well, but I realize now that most folks will know exactly which output they have (either a binary blob or json format) and so can explicitly do the casting themselves. Changed so it's not exported.